### ISSUE TITLE
What symbols are needed for REPL outputs?

### ISSUE THREAD

**Issue by kescobo** [tlienart/Xranklin.jl#242]
![image](https://github.com/tlienart/Xranklin.jl/assets/3502975/1e3e1a58-d2d7-4f59-8595-4b2ee8bfe153)

I thought that my terminal was using font with all the symbols I might need, but I'm getting this output. Just curious if there's a recommended font or a list of used glyphs

**Comment by kescobo** [comment]
I really love the emojis though, don't get rid of them!

**Comment by kescobo** [comment]
>Hahaha so much for using emojis in info stuff 😱 ; what you boxed in red should have shown as 1️⃣ . 

What happens when I try to make `Char`s out of those two... maybe this can help somehow?

![image](https://github.com/tlienart/Xranklin.jl/assets/3502975/429a453e-514d-45d0-af61-cc3a3426f6fa)

**Comment by tlienart** [comment]
Right, so it looks like 1️⃣ is not part of the base emoji symbols here: https://github.com/JuliaLang/julia/blob/master/stdlib/REPL/src/emoji_symbols.jl whereas things like the hourglass (` "\\:hourglass_flowing_sand:" => "⏳",`) or the lightbulb (`"\\:bulb:" => "💡",`) are. So probably the safest is to only use emojis from that list.
